driver/format/RowBinaryWithNamesAndTypes.cpp

```diff
@@ -152,14 +153,24 @@ void RowBinaryWithNamesAndTypesResultSet::readValue(WireTypeDateTimeAsInt & dest
     readPOD(dest.value);
 }
 
+void RowBinaryWithNamesAndTypesResultSet::readValue(WireTypeDateTime64AsInt & dest, ColumnInfo & column_info) {
+    readPOD(dest.value);
```

Comment:
Did you forget to use\convert value according to a timezone?

Reply:
It should be done later on the call stack (`TODO` yes, not the goal of this PR):

https://github.com/traceon/clickhouse-odbc/blob/67b36ef86ebea0fd5372b219f5c84c9771b1eacc/driver/utils/type_info.h#L1871-L1901